_posts/2023-06-28-Remilia-interview.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Interview wth contributor Remilia Scarlet"
+author: beta-zilliani,Remilia
+summary: "<b>Music playback projects in Crystal, games engines, and a bit of Lisp"
+---
+<https://remilia.sdf.org/>
+
+<https://www.youtube.com/watch?v=UX2in-whUik>
```

Comment:
Is there a specific reason to put these on top? I would usually expect further reading links at the bottom.